concourse/pipelines/templates/resources/release-artifact-tpl.yml

```
@@ -0,0 +1,17 @@
+{# Template for a PXF build artifact resource definition.
+ #
+ # Expects a dictionary with the name 'x' to be set as defined in macros.j2 file
+ #}
+
+{# include Release artifact only if both build and test platform are the same to avoid duplicates #}
+{% if x.build_platform == x.test_platform %}
+
+{# Release Drop Artifact #}
+- name: pxf-gp[[x.gp_ver]]-[[x.test_platform]]-release-to-releng
+  type: gcs
+  source:
+    bucket: ((ud/pxf/[[environment]]/releng-drop-bucket-name))
+    json_key: ((concourse-gcs-resources-service-account-key))
+    regexp: ((ud/pxf/common/releng-drop-path))/gpdb[[x.gp_ver]]/[[x.pxf_release_package_filename_regex]]
+
+{% endif %}  {# end of conditional package definitions #}
```

```suggestion
{% endif %}{# end of conditional package definitions #}
```
this space might have nasty side effects, just remove it